### DISCUSSION TITLE
Change listening port

### DISCUSSION THREAD

**Discussion by stanthewizzard** [ZoeyVid/NPMplus#929] [Q&A] [answered]
Hello
I'm able to change the admin port but I want also to change HTTP and HTTPS port.
Is there a way to do that ?

Thanks

**Comment by Zoey2936** [comment, marked as answer]
Currently not, but you can disable http listining at all, why do you want to change the ports?

**Reply by stanthewizzard** [reply]
npmplus container is on the same host of npm container
443 is already used
I want to test and migrate

**Reply by Zoey2936** [reply]
For testing you could install it on docker desktop
